README.md

```diff
@@ -1,2 +1,164 @@
 # github-backup
 :back: :arrow_up: A script to backup your GitHub Account/Organisation so you can still work when (they/you are) offline.
+
+## Why?
+
+As a person who uses GitHub as their "_single-source-of-truth_"
+I need a backup of GitHub issues
+so that I can work "offline".
+(_either when I have no network or GH is "down"_)
+
+## What?
+
+Backup GitHub issues.
+
++ a script that lets you login to GitHub using OAuth
++ Use GitHub REST API to retrieve all issues
++ Store issues as "flat files" in subdirectories corresponding to the GitHub URL
+e.g: if the issue is `dwyl/github-reference/issues/15`
+store it as `dwyl/github-reference/issues/15.json`
++ It does not need to "scale" it just needs to work. read: http://paulgraham.com/ds.html
+
+Note: initially we do not need a Web-UI to let people backup their issues,
+but later we could build UI to make it more useable.
+
+## How?
+
++ Elixir
++ Phoenix
++ Tachyons
+
+## Set up
+
+The project is hosted on heroku at: https://github-backup.herokuapp.com/
+
+To run it locally, you will need to:
+
+- Create a new Github application.
+- Run a phoenix server on your machine.
+
+You'll need to have installed [Elixir](https://elixir-lang.org/install.html), [Phoenix](https://hexdocs.pm/phoenix/installation.html), and [ngrok](https://ngrok.com/download) if you haven't already.
```

Comment:
please separate these prerequisites into a bullet-point list and _explain_/_clarify_ why ngrok.com is needed.